apps/src/code-studio/progressRedux.js

```diff
@@ -534,6 +534,7 @@ const lessonFromLesson = lesson =>
     'id',
     'lockable',
     'lessonNumber',
+    'lessonStartUrl',
```

Comment:
Personally this name does not make it completely clear to be what this url is for. This link is for students right? Could we include that in this variable? 'studentLessonStartUrl' or 'lessonStartLocationStudent'.... there might be other options I'm not thinking of but I think making it clear this is specific to students would be good.

Also as I'm saying that I'm thinking about the changes we are going to make to use "participant" instead of student. It could be interesting to try to start using that language here. So maybe 'participantLessonStartUrl'

Reply:
I'm not sure that I agree that it matters who the link is for. We are currently using it in the "send to students" button, but I think it also represents the correct "starting point" for a lesson regardless of who is looking at it. I do like this switch in language to "participant" though!

Reply:
Is it the URL for teachers to start the lesson too?

Reply:
It's the url to the first page of the lesson, so would also make sense as a starting place for teachers if they were looking to demo or preview- is that what you mean?

Reply:
I see what you are saying its the place to start the lesson if you are progressing through it. I was thinking a teacher would start at the lesson plan but I take back my concern about the name after this discussion. Thanks for talking it through!

Reply:
Of course! Thanks for your feedback on this!